src/widgets/RealtimeEffectPanel.cpp

```diff
@@ -0,0 +1,813 @@
+            window->Raise();
+      });
+      Bind(EVT_MOVABLE_CONTROL_DRAG_POSITION, [this, dropHintLine](const MovableControlEvent& event)
+      {
+         constexpr auto DropHintLineHeihgh { 3 };//px
+
+         auto sizer = mEffectListContainer->GetSizer();
+         assert(sizer != nullptr);
+
+         if(event.GetSourceIndex() == event.GetTargetIndex())
+            //do not display hint line if position didn't change
+            dropHintLine->SetSize({0, 0});
+         else if(event.GetTargetIndex() == sizer->GetItemCount())
+         {
+            //insert at end
+            auto lastItem = sizer->GetItem(sizer->GetItemCount() - 1);
+            //avoid clipping
+            dropHintLine->SetPosition(lastItem->GetRect().GetBottomLeft() - wxPoint(0, DropHintLineHeihgh));
+            dropHintLine->SetSize(lastItem->GetSize().x, 3);
+         }
+         else if(auto item = sizer->GetItem(event.GetTargetIndex()))
+         {
+            dropHintLine->SetPosition(item->GetRect().GetTopLeft());
+            dropHintLine->SetSize(item->GetSize().x, 3);
+         }
+      });
+      Bind(EVT_MOVABLE_CONTROL_DRAG_FINISHED, [this, dropHintLine](const MovableControlEvent& event)
+      {
+         dropHintLine->Hide();
+         
+         auto trackOwner = mTrack->GetOwner();
+         if(!trackOwner)
+            return;
+
+         auto project = trackOwner->GetOwner();
+         if(project == nullptr)
+            return;
+
+         auto& effectList = RealtimeEffectList::Get(*mTrack);
+         auto from = event.GetSourceIndex();
+         auto to = event.GetSourceIndex() < event.GetTargetIndex() ? event.GetTargetIndex() - 1 : event.GetTargetIndex();
+         if(from != to)
+         {
+            effectList.MoveEffect(from, to);
+            ProjectHistory::Get(*project).PushState(
+               //i18n-hint: undo history, first parameter - track name
+               XO("'%s' effects reorder %d->%d").Format(mTrack->GetName(), from, to),
+               //i18n-hint: undo history record
+               XO("Realtime effect reorder"), UndoPush::CONSOLIDATE);
+         }
+         else
+         {
+            wxWindowUpdateLocker freeze(this);
+            Layout();
+         }
+      });
+      SetScrollRate(0, 20);
+   }
+
+   void OnEffectListItemChange(const RealtimeEffectListMessage& msg)
+   {
+      wxWindowUpdateLocker freeze(this);
+      if(msg.type == RealtimeEffectListMessage::Type::Move)
+      {
+         const auto sizer = mEffectListContainer->GetSizer();
+
+         const auto movedItem = sizer->GetItem(msg.index);
+
+         const auto proportion = movedItem->GetProportion();
+         const auto flag = movedItem->GetFlag();
+         const auto border = movedItem->GetBorder();
+
+         auto window = movedItem->GetWindow();
+         
+         sizer->Remove(msg.index);
+         sizer->Insert(std::any_cast<size_t>(msg.extra), window, proportion, flag, border);
```

Comment:
This breaks the macOS build.  Only the pointer-valued overload of `any_cast` is available, with our minimum targeted version.